runtime-testsuite/pom.xml

```diff
@@ -106,6 +106,8 @@
                         <include>**/dart/Test*.java</include>
                         <include>${antlr.tests.swift}</include>
                     </includes>
+          <parallel>all</parallel>
+          <useUnlimitedThreads>true</useUnlimitedThreads>
```

Comment:
Please fix the unaligned indentation.

Reply:
Done. Editor uses spaces by default, file was using tabs.